src/classnames/tailwind.ts

```diff
@@ -7384,8 +7391,10 @@ export type TWidth =
   | 'w-thin-mobile'
   | 'w-thin-card'
   | 'w-card'
+  | 'w-app-card'
   | 'w-mobile-card'
   | 'w-full-125'
+  | 'w-full-105'
```

Comment:
Note that this block is a little wider it's parent
![image](https://user-images.githubusercontent.com/49658988/187157620-39d04cca-4810-41be-9f6c-36eaf9644794.png)